x-pack/solutions/observability/test/api_integration_deployment_agnostic/apis/observability_agent/utils/one_chat_client.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  ChatRequestBodyPayload,
+  ChatResponse,
+} from '@kbn/onechat-plugin/common/http_api/chat';
+import type { Conversation } from '@kbn/onechat-common';
+import type { SupertestWithRoleScopeType } from '../../../services';
+
+export function createOneChatApiClient(supertest: SupertestWithRoleScopeType) {
```

Comment:
nit: might be better to use the "agent builder" term because we will eventually rename (I think). But it'll be inconsistent in the interim regardless what you choose so not a big deal.

```suggestion
export function createAgentBuilderClient(supertest: SupertestWithRoleScopeType) {
```

Reply:
Makes sense. Thanks @sorenlouv 
Updated - https://github.com/elastic/kibana/pull/242482/commits/b0f687fb76d79c5b41cea8f38c9e167cada3c81a